openpype/settings/entities/schemas/projects_schema/schemas/schema_maya_publish.json

```diff
@@ -129,6 +129,62 @@
             ]
         },
 
+        {
+            "type": "dict",
+            "collapsible": true,
+            "key": "ValidateMayaUnits",
+            "label": "Validate Maya Units",
+            "checkbox_key": "enabled",
+            "children": [
+                {
+                    "type": "boolean",
+                    "key": "enabled",
+                    "label": "Enabled"
+                },
+                {
+                    "type": "boolean",
+                    "key": "validate_linear_units",
+                    "label": "Validate linear units"
+                },
+                {
+                          "key": "linear_units",
+                          "label": "Linear units",
+                          "type": "enum",
+                          "multiselection": false,
+                          "defaults": "cm",
+                          "enum_items": [
+                            {"mm": "millimeter"},
+                            {"cm": "centimeter"},
+                            {"m":  "meter"},
+                            {"in":  "inch"},
+                            {"ft": "foot"},
+                            {"yd": "yard"}
```

Comment:
Not saying we REALLY need them all but it does seem like [the Maya command has more possible settings for linear units](https://download.autodesk.com/us/maya/2011help/Commands/currentUnit.html#flaglinear).

> Valid strings are:
> [mm | millimeter | cm | centimeter | m | meter | km | kilometer | in | inch | ft | foot | yd | yard | mi | mile]

Which means we are missing **(km) kilometer** and **(mi) mile**.
_(Not that one should ever mess with Maya's default units. But hey, some people like to be daredevils!)_

Reply:
strange thing as Maya (at least 2020) is not exposing them from UI:
![image](https://user-images.githubusercontent.com/33513211/153080829-03a4254b-4c31-4025-95d1-4d83e1c5453c.png)

Reply:
Even they weren't confident enough. I guess then we should ignore the other options. I'll mark this resolved.